src/web/components/SharingPermission/SharingPermissionsTable.tsx

```diff
@@ -3,41 +3,59 @@ import { CheckedState } from '@radix-ui/react-checkbox';
 import { useEffect, useMemo, useState } from 'react';
 
 import { ParticipantTypeDTO } from '../../../api/entities/ParticipantType';
-import { AvailableParticipantDTO } from '../../../api/routers/participantsRouter';
+import { useAvailableSiteList } from '../../services/site';
+import { formatStringsWithSeparator } from '../../utils/textHelpers';
 import { Dialog } from '../Core/Dialog';
+import { Loading } from '../Core/Loading';
 import { MultiSelectDropdown } from '../Core/MultiSelectDropdown';
 import { SortableTableHeader } from '../Core/SortableTableHeader';
 import { TriStateCheckbox, TriStateCheckboxState } from '../Core/TriStateCheckbox';
+import { isAddedByManual, MANUALLY_ADDED, SharingParticipant } from './bulkAddPermissionsHelpers';
 import { ParticipantsTable } from './ParticipantsTable';
 
 import './SharingPermissionsTable.scss';
 
 type SharingPermissionsTableProps = {
-  sharingParticipants: AvailableParticipantDTO[];
+  sharedSiteIds: number[];
+  sharedTypes: string[];
   onDeleteSharingPermission: (siteIds: number[]) => Promise<void>;
   participantTypes: ParticipantTypeDTO[];
 };
 
-function NoParticipant() {
```

Comment:
Was this actually being used before? I know we have conditions to hide the whole collapsible, but I think it would be weird for the whole table to disappear if a user removed all their existing sharing permissions, so I'd argue for keeping this function for that case. Also, this is was Kristen said in Figma:
![image](https://github.com/IABTechLab/uid2-self-serve-portal/assets/137864392/1c76792a-4fab-4677-a3d1-b418b0311ece)

Reply:
I saw we have condition  `sharedSiteIds.length > 0 || sharedTypes.length > 0` on the table, so I think in this case, the table will always have participants showing? If both sharedSiteIds and sharedTypes I think we are showing recommendation again? should we show this section as well?

Reply:
I'm actually not sure of the best way to handle this. I think maybe the ideal state is the we hide the table when the user first sees this screen (e.g. before they have gone through the recommendations workflow), but then always show this section after that. That way, if the user removes all existing permissions then the section won't just disappear. 

> if both sharedSiteIds and sharedTypes I think we are showing recommendation again?
Actually I am going to change the condition for recommendation to just be `sharedTypes.length === 0`, if that is any help. Otherwise the checkbox was misleading as it looks like "DSP" is already saved, even though it is just recommended. I think it makes more sense to just be `sharedTypes.length === 0`.

Reply:
<img width="1495" alt="image" src="https://github.com/IABTechLab/uid2-self-serve-portal/assets/116539741/4af1effc-e9cb-4a30-ab33-15cbb85373cf">
I added this back, but the user won't to be able to see this state for now. I think once we have that recommendation flag, we could then hide the table from user who haven't gone through recommendations before.

Reply:
yep good idea!